contribute/document-preview.md

```diff
@@ -60,6 +60,7 @@ By default, a browser window will open at [http://localhost:3000](http://localho
 
 ![alt_text](media/website-preview.png)
 
+When you click on Docs, you will be taken to the latest stable version (e.g. `http://localhost:3000/docs/2.10.x/`).  If you're previewing `master` which is current, then you need to change the url to `http://localhost:3000/docs/next`
```

Comment:
```suggestion
::: tip
When you click on `Docs`, you are taken to the latest stable version (e.g., `http://localhost:3000/docs/2.10.x/`). If you want to preview changes on `master`, change the URL to `http://localhost:3000/docs/next`
:::
```
Write in the simple present tense as much as possible if you are covering facts that were, are, and forever shall be true. https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit?pli=1#bookmark=id.e8uqh1awkcnp